PRIVACY.md

```diff
@@ -2,7 +2,7 @@
 
 This is a plain English summary of all of the components within Ghost which may affect your privacy in some way. Please keep in mind that if you use third party Themes or Apps with Ghost, there may be additional things not listed here.
 
-Each of the items listed in this document can be disabled via Ghost's `config.js` file. Check out the [configuration guide](https://docs.ghost.org/v1.0.0/docs/configuring-ghost) for details.
+Each of the items listed in this document can be disabled via Ghost's `config.[env].json` file. Check out the [configuration guide](https://docs.ghost.org/v1.0.0/docs/config) for details.
```

Comment:
can you use this url instead? https://docs.ghost.org/docs/config#section-privacy It's a bit more specific